cmd/drone-server/config/config.go

```diff
@@ -293,9 +293,12 @@ type (
 
 	// Gitea provides the gitea client configuration.
 	Gitea struct {
-		Server     string `envconfig:"DRONE_GITEA_SERVER"`
-		SkipVerify bool   `envconfig:"DRONE_GITEA_SKIP_VERIFY"`
-		Debug      bool   `envconfig:"DRONE_GITEA_DEBUG"`
+		Server       string   `envconfig:"DRONE_GITEA_SERVER" default:"https://try.gitea.io"`
```

Comment:
this default needs to be removed. An empty string is the only way we know if gitea is configured or not.

Reply:
Good catch. Thanks :)

I've updated.